classes/DirtEventTypes.sc

```diff
@@ -108,15 +109,15 @@ DirtEventTypes {
 
 				if (hasNote) {
 					freq = ~freq.value;
-					~midinote = (freq.cpsmidi).round(1).asInteger;
+					~midinote = ((freq.cpsmidi).round(1) ? 0).asInteger;
```

Comment:
the default is not necessary and can be dropped. `freq.cpsmidi).round(1)` will never be `nil`.